albumentations/augmentations/transforms.py

```diff
@@ -3550,3 +3553,61 @@ def apply(self, img, transforms=(), **params):
 
     def get_transform_init_args_names(self):
         return ("brightness", "contrast", "saturation", "hue")
+
+
+class Superpixels(ImageOnlyTransform):
+    """Completely or partially transform the input image to its superpixel representation. Uses skimage's version
+    of the SLIC algorithm. May be slow.
+    Args:
+        p_replace (float): defines the probability of any superpixel area being replaced by the superpixel, i.e. by
+            the average pixel color within its area. Default: 0.1.
+        n_segments (int): target number of superpixels to generate. Default: 100.
+        p (float): probability of applying the transform. Default: 0.5.
+    Targets:
+        image
+    """
+
+    def __init__(self, p_replace=0.1, n_segments=100, always_apply=False, p=0.5):
```

Comment:
Please give opportunity to set `n_segments` and `p_replace` as ranges.
Also add `max_size` argument support.

Reply:
why do we need n_segments and p_replace as ranges?

Reply:
Because if we set them as ranges we can randomly sample them. In imgaug we can set them as ranges, and in our wrapper we also can set them as ranges as well.